job-descriptions/head-of-customer-success.md

```diff
@@ -0,0 +1,71 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Head of Customer Success
+
+### About us
+
+Sourcegraph's mission is to enable every software developer to create products using the best technology. The innovations of the future will all rely on software, and we want to bring the future sooner by building the best tools that help developers answer questions about their code. You can learn more about our mission by reading [our master plan](https://sourcegraph.com/plan).
+
+Our values:
+
+- **Inclusivity**. To build a product that serves the needs of all developers, we are building a diverse team that is distributed across the world. Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
```

Comment:
These are good things but they aren't our values (https://about.sourcegraph.com/company/values).

Reply:
Thanks, I had an outdated template in here